src/BizHawk.Emulation.Cores/BizHawk.Emulation.Cores.csproj

```diff
@@ -5,6 +5,7 @@
 		<TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
 	</PropertyGroup>
 	<ItemGroup>
+		<PackageReference Include="ELFSharp" Version="2.10.0" />

```

Comment:
Get rid of penet

Reply:
Is this a TODO, or is PeNet made redundant by this PR?

Reply:
PeNet is made redundant by this PR, so I am going to yoink it.  PeWrapper is gone.